### ISSUE TITLE
The number of training sets cannot be less than the validation set.

### ISSUE THREAD

**Issue by oneforcat** [kerlomz/captcha_trainer#91]

**Comment by coder-gao** [comment]
我想知道你训练了几张图片啊

**Comment by oneforcat** [comment]
> 我想知道你训练了几张图片啊

我现在去搞几千张试试

**Comment by oneforcat** [comment]
> 我想知道你训练了几张图片啊

大概要多少张才够啊，还有就是，gui版本的操作流程是怎么样的？

**Comment by coder-gao** [comment]
> > 我想知道你训练了几张图片啊
> 
> 大概要多少张才够啊，还有就是，gui版本的操作流程是怎么样的？
https://www.bilibili.com/video/BV1zT4y1j7GQ?from=search&seid=14194506346371061832

**Comment by xuebin-git** [comment]
我训练510张图片，也报这个错误

**Comment by xuebin-git** [comment]
GUI流程是按照教程做的，是对图片数量有要求吗

**Comment by kerlomz** [comment]
字面意思，训练集要超过验证集，验证集没有配置的情况下默认是从训练集中随机抽300张

**Comment by xuebin-git** [comment]
INFO:tensorflow:Loading Validation DataSet...
INFO:tensorflow:Total 210 Trains DataSets
INFO:tensorflow:Total 300 Validation DataSets

**Comment by xuebin-git** [comment]
训练集为啥只有210呢，我一共510张图片
是不是要大于600张图片？

**Comment by kerlomz** [comment]
建议一共2000张图片以上
